tests/test_single_topology.py

```diff
@@ -739,15 +739,15 @@ def setup_intermediate_nonbonded_term_ref(src_nonbonded, dst_nonbonded, lamb, al
 
     for lamb in np.linspace(0.0, 1.0, 10):
         nonbonded_ref = setup_intermediate_nonbonded_term_ref(
-            st.src_system.nonbonded,
-            st.dst_system.nonbonded,
+            st.src_system.nonbonded_pair_list,
```

Comment:
I'm hoping this change makes it easier for the reader to be able to quickly infer if potential type for `src_system.nonbonded` is one of {pair_list, all_pairs, ixn_group}